### PR TITLE
Migrate the Quality Patches Tool home page to Experience League

### PR DESCRIPTION
**Pull request by barnato-v**
## Purpose of this pull request

This pull request (PR) is to add meta data about migration of this page to Adobe Experience League Commerce Operations 

## Affected DevDocs pages

https://devdocs.magento.com/quality-patches/tool.html

whatsnew
The Quality Patches Tool [home page](https://devdocs.magento.com/quality-patches/tool.html) moved to [Experience League](https://experienceleague.adobe.com/tools/commerce-quality-patches/index.html) and will be redirected soon.

### CODE REVIEW

File: src/quality-patches/tool.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ group: tools
 title: Quality Patches Tool
 functional_areas:
   - Upgrade
+redirect_from: https://devdocs.magento.com/quality-patches/tool.html
```
Comment (jeff-matthews) [inline src/quality-patches/tool.md:6]: No need for this redirect here. Please remove.

File: src/quality-patches/tool.md [hunk 2]
```diff
@@ -3,6 +3,9 @@ group: tools
 title: Quality Patches Tool
 functional_areas:
```
Comment (jeff-matthews) [inline src/quality-patches/tool.md:4]:
```suggestion
group: tools
functional_areas:
```

Comment (jeff-matthews) [reply src/quality-patches/tool.md:4]: This is required when migration metadata is present. The generator that creates the [changelog](https://devdocs.magento.com/migrated.html) of migrated topics uses it as a required parameter. Tests will fail without it.

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

**Review by jeff-matthews — APPROVED** [review]

**Comment by jeff-matthews** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@jeff-matthews the branch with the code is successfully imported into the `magento-devdocs/devdocs` repository. Branch name: `imported-barnato-v-devdocs-9521`.

**Pull request merged**